### DISCUSSION TITLE
Require information on protractor-cypress migration

### DISCUSSION THREAD

**Discussion by nikhithavangaru** [cypress-io/cypress#22469] [General]
Hi
I am Lakshmi Nikhitha working at one of the big reputed companies, I would like to know few things that,

1. We are planning to move all the scripts from protractor to cypress. We would like to know what are the requirements, possibilities and how to implement them.

2. Is there any paid version available for cypress which supports additional features such as multi-tabs, X-Path support, remote execution etc.?

Please let me know for any further information. 

Thanks & Regards
Lakshmi Nikhitha Vangaru

**Comment by nikhithavangaru** [comment]
Hi 

please let me know any kind of response regarding the above query.

Thank you

**Comment by jjhelguero** [comment]
I cannot speak for your first question.

For your second one:
There are ways to overcome using multi-tabs with Cypress. Here are a few [examples](https://glebbahmutov.com/blog/cypress-second-tab/).
If you must use XPath, there is a [plugin](https://github.com/cypress-io/cypress-xpath) for that.
[Cypress Dashboard](https://www.cypress.io/dashboard/) is a great tool for parallelized execution with your CI of choice along with analytics and many other benefits in the long term.

**Comment by nikhithavangaru** [comment]
Hi

Thanks for the response,
can anyone let me know, what are the paid versions available for cypress and let me know what are the benifits and features available for that.


Thank you..
